metadatastore/commands.py

```diff
@@ -1165,3 +1165,46 @@ def find_last(num=1):
         if next(c) == num:
             raise StopIteration
         yield _cache_run_start(rs)
+
+
+def get_header(run_start, allow_no_run_stop):
```

Comment:
The question is, what db[] should return? The current answer (headers as
implemented) is not a bad one, but it has its problems. I suspect it is
trying to be too many things at once. Let's meet in person next week to
evaluate alternatives.
On Thu, Sep 10, 2015 at 7:04 PM dchabot <notifications@github.com> wrote:

> In metadatastore/commands.py
> <https://github.com/NSLS-II/metadatastore/pull/169#discussion_r39225337>:
>
> > @@ -1165,3 +1165,46 @@ def find_last(num=1):
> >          if next(c) == num:
> >              raise StopIteration
> >          yield _cache_run_start(rs)
> > +
> > +
> > +def get_header(run_start, allow_no_run_stop):
>
> On the surface, this sounds like a recipe for confusion...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/metadatastore/pull/169/files#r39225337>.
>